### DISCUSSION TITLE
Cape in Prism Launcher

### DISCUSSION THREAD

**Discussion by Pressslyn** [PrismLauncher/PrismLauncher#3210] [Q&A]
I just finished the requirements to get the new cape " Eyeblossom " and i cant see it in game even if i enabled in the launcher, even updated it on minecraft.net
![image](https://github.com/user-attachments/assets/16bbd6a0-8085-4ae2-aa14-36b5454aea2d)
![image](https://github.com/user-attachments/assets/98b5747a-833d-4e54-9bbf-a0ad1c564573)
I checked others with the same problem, which they stated that when they change the skin / reselect the cape it'll work.
But when i tried it, it didnt. ( I even re-registered the account in Prism Launcher )

**Comment by eoFlashh** [comment]
1 check in namemc if it is actually equipped
2 check if the skin is not disabled in the skin customization menu
3 if none of these options work, load a skin and select the skin you want to use in Prism Launcher (it may solve the problem)
![image](https://github.com/user-attachments/assets/a0430d74-64f8-4fbe-bc38-58b91e1d2878)
